src/commands/workspace/refreshWorkspace.ts

```diff
@@ -0,0 +1,11 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { ext } from "../../extensionVariables";
+
+export async function refreshWorkspace(context: IActionContext): Promise<void> {
+    await ext.workspaceTree.refresh(context);
```

Comment:
Not sure if we really need a whole command for this line, but if we keep this, you should use it here https://github.com/microsoft/vscode-azureresourcegroups/pull/124/files?diff=split&w=1#diff-9cb9d4ec34a86167c68d8cb55f3d8de6b90eadf242b13a1f2e949608b1ea9401R21

Reply:
Would you prefer putting the function body directly in the registerCommand arguments?